applications/app/controllers/CrosswordsController.scala

```diff
@@ -51,7 +51,7 @@ trait CrosswordController extends Controller with Logging with ExecutionContexts
 
 class CrosswordPageController(val contentApiClient: ContentApiClient) extends CrosswordController {
 
-  def noResults()(implicit request: RequestHeader) = InternalServerError("Content API query returned an error.")
+  def noResults()(implicit request: RequestHeader) = NoCache(NotFound)
```

Comment:
You might want to cache the response, even for a very small amout of time to avoid ddos `Cached(CacheTime.NotFound)(NotFound)`